src/Data/Bifunctor/Biap.hs

```diff
@@ -10,6 +10,10 @@
 {-# LANGUAGE DeriveGeneric              #-}
 #endif
 
+-- This module uses GND
+#if __GLASGOW_HASKELL__ >= 702
```

Comment:
This could be combined with the existing `#if __GLASGOW_HASKELL__ >= 702 ... #endif` above.

Reply:
I'd rather keep them separate. *If* GND eventually becomes Safe :)